internals/core_enums.py

```diff
@@ -0,0 +1,283 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+
+
+WEBCOMPONENTS = 1
+MISC = 2
+SECURITY = 3
+MULTIMEDIA = 4
+DOM = 5
+FILE = 6
+OFFLINE = 7
+DEVICE = 8
+COMMUNICATION = 9
+JAVASCRIPT = 10
+NETWORKING = 11
+INPUT = 12
+PERFORMANCE = 13
+GRAPHICS = 14
+CSS = 15
+HOUDINI = 16
+SERVICEWORKER = 17
+WEBRTC = 18
+LAYERED = 19
+WEBASSEMBLY = 20
+CAPABILITIES = 21
+
+FEATURE_CATEGORIES = {
+  CSS: 'CSS',
+  WEBCOMPONENTS: 'Web Components',
+  MISC: 'Miscellaneous',
+  SECURITY: 'Security',
+  MULTIMEDIA: 'Multimedia',
+  DOM: 'DOM',
+  FILE: 'File APIs',
+  OFFLINE: 'Offline / Storage',
+  DEVICE: 'Device',
+  COMMUNICATION: 'Realtime / Communication',
+  JAVASCRIPT: 'JavaScript',
+  NETWORKING: 'Network / Connectivity',
+  INPUT: 'User input',
+  PERFORMANCE: 'Performance',
+  GRAPHICS: 'Graphics',
+  HOUDINI: 'Houdini',
+  SERVICEWORKER: 'Service Worker',
+  WEBRTC: 'Web RTC',
+  LAYERED: 'Layered APIs',
+  WEBASSEMBLY: 'WebAssembly',
+  CAPABILITIES: 'Capabilities (Fugu)'
+}
+
+FEATURE_TYPE_INCUBATE_ID = 0
+FEATURE_TYPE_EXISTING_ID = 1
+FEATURE_TYPE_CODE_CHANGE_ID = 2
+FEATURE_TYPE_DEPRECATION_ID = 3
+
+FEATURE_TYPES = {
+  FEATURE_TYPE_INCUBATE_ID: 'New feature incubation',
+  FEATURE_TYPE_EXISTING_ID: 'Existing feature implementation',
+  FEATURE_TYPE_CODE_CHANGE_ID: 'Web developer facing change to existing code',
+  FEATURE_TYPE_DEPRECATION_ID: 'Feature deprecation',
+}
+
+
+# Intent stages and mapping from stage to stage name.
+INTENT_NONE = 0
+INTENT_INCUBATE = 7  # Start incubating
```

Comment:
I would like to understand the naming convention. There's no such thing as an intent to incubate. Same with removed, shipped, and parked.

Reply:
This PR just shoveled the existing enums from models.py into a new file.   This PR is just showing what file they would be located in.  

I would like to redo these enums to give them better names and make them more comprehensive for all our processes.  That will come later, but you are right to point it out so that it does not get forgotten.